snuba/datasets/configuration/json_schema.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import annotations
+
+from typing import Any
+
+TYPE_STRING = {"type": "string"}
+
+stream_loader_schema: Any = {
+    "type": "object",
+    "properties": {
+        "processor": TYPE_STRING,
+        "default_topic": TYPE_STRING,
+        "commit_log_topic": TYPE_STRING,
+        "subscription_scheduled_topic": TYPE_STRING,
+        "subscription_scheduler_mode": TYPE_STRING,
+        "subscription_result_topic": TYPE_STRING,
+        "replacement_topic": {"type": ["string", "null"]},
+        "prefilter": {
+            "type": "object",
+            "properties": {
+                "type": TYPE_STRING,
+                "args": {"type": "array", "items": TYPE_STRING},
+            },
+        },
+        "dlq_policy": {
+            "type": "object",
+            "properties": {
+                "type": TYPE_STRING,
+                "args": {"type": "array", "items": TYPE_STRING},
+            },
+        },
+    },
+}
+
+######
+# Column specific json schemas
+def make_column_schema(column_type: Any, args: Any) -> Any:
+    props = {
+        "name": TYPE_STRING,
+        "type": column_type,
+        "schema_modifiers": {"type": "array", "items": TYPE_STRING},
+    }
+    if args is not None:
+        props["args"] = args
+    return {
+        "type": "object",
+        "properties": props,
+    }
+
+
+Number_schema: Any = make_column_schema(
+    column_type={"enum": ["UInt", "Float"]},
+    args={"type": "object", "properties": {"size": {"type": "number"}}},
+)
+
+
+No_arg_schema = make_column_schema(
```

Comment:
nit: casing here is weird, i'd be fine with `NO_ARG_SCHEMA` or `no_arg_schema` (or maybe `NoArgSchema`?)

Reply:
I agree, constants should be capital case.